react/features/base/conference/functions.ts

```diff
@@ -257,11 +256,11 @@ export function getConferenceOptions(stateful: Function | Object) {
 /**
 * Returns the UTC timestamp when the first participant joined the conference.
 *
-* @param {Function | Object} stateful - Reference that can be resolved to Redux
+* @param {IStateful} stateful - Reference that can be resolved to Redux
 * state with the {@code toState} function.
 * @returns {number}
 */
-export function getConferenceTimestamp(stateful: Function | Object): number {
+export function getConferenceTimestamp(stateful: IStateful) {
```

Comment:
Why did you drop the return type annotation?

Reply:
Because it can also return `undefined`, and instead of writing `number | undefined` I chose to remove it because TypeScript can figure it out just fine.

Reply:
Ack